background/sync-manager.js

```diff
@@ -223,11 +218,12 @@ const syncMan = (() => {
   }
 
   function getErrorBadge() {
-    if (status.state === STATES.connected && lastError && !isNetworkError(lastError)) {
+    if (status.state === STATES.connected &&
+        (!status.login || lastError && !isNetworkError(lastError))) {
       return {
         text: 'x',
         color: '#F00',
-        title: isGrantError(lastError) ? 'syncErrorRelogin' : 'syncError',
+        title: !status.login ? 'syncErrorRelogin' : 'syncError',
```

Comment:
```suggestion
        title: status.login ? 'syncError' :  'syncErrorRelogin',
```

Reply:
Looks like an attempt to adhere to [no-negated-condition](https://eslint.org/docs/rules/no-negated-condition) eslint rule. Don't propose such cosmetic changes, it only slows down development. The proper way for you to become a permanent contributor and propose PRs that handle such issues globally but first we all should discuss in a new issue if we want a particular cosmetic change.

Reply:
I was not trying to adhere to a particular linting rule, only to avoid an unnecessary negation of `status.login`, but your explanation in the last paragraph makes sense. My code review was aimed to help, not to impose cosmetic changes.